src/features/content-answers/ContentAnswersOpenButton.scss

```diff
@@ -0,0 +1,24 @@
+@import '../../styles/variables';
+
+.bdl-ContentAnswersOpenButton {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: 40px;
+    height: 40px;
+    background-color: $bdl-gray-02;
+    border: none;
+
+    .btn-content {
+        display: flex;
+    }
+
+    &:focus {
+        background-color: $bdl-gray-10;
+    }
+
+    &:not(.bdl-is-disabled):hover,
+    &:not(.is-disabled):hover {
```

Comment:
it's probably safe to remove the `.is-disabled` as we'd like to migrate uses to have the `bdl` prefix